aspnetcore/blazor/security/index.md

```diff
@@ -0,0 +1,482 @@
+| Work or School Accounts<br>Organizational authentication for a single tenant.            | `SingleOrg`              |
+| Work or School Accounts<br>Organizational authentication for multiple tenants.           | `MultiOrg`               |
+| Windows Authentication                                                                   | `Windows`                |
+
+The command creates a folder named with the value in the `{APP NAME}` placeholder and uses the same value as the app name. For more information, see the [dotnet new](/dotnet/core/tools/dotnet-new) command in the .NET Core Guide.
+
+Open the project's folder in Visual Studio Code.
+
+<!--
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+1. Follow the Visual Studio for Mac guidance in the <xref:blazor/get-started> article.
+
+1.
+
+1.
+
+-->
+
+   # [.NET Core CLI](#tab/netcore-cli/)
+
+Follow the .NET Core CLI guidance in the <xref:blazor/get-started> article to create a new Blazor server-side project with an authentication mechanism:
+
+```console
+dotnet new blazorserverside -o {APP NAME} -au {AUTHENTICATION}
+```
+
+Permissible authentication values (`{AUTHENTICATION}`) are shown in the following table.
+
+| Authentication mechanism                                                                 | `{AUTHENTICATION}` value |
+| ---------------------------------------------------------------------------------------- | :----------------------: |
+| No Authentication                                                                        | `None`                   |
+| Individual<br>Users stored in the app with ASP.NET Core Identity.                        | `Individual`             |
+| Individual<br>Users stored in [Azure AD B2C](xref:security/authentication/azure-ad-b2c). | `IndividualB2C`          |
+| Work or School Accounts<br>Organizational authentication for a single tenant.            | `SingleOrg`              |
+| Work or School Accounts<br>Organizational authentication for multiple tenants.           | `MultiOrg`               |
+| Windows Authentication                                                                   | `Windows`                |
+
+The command creates a folder named with the value in the `{APP NAME}` placeholder and uses the same value as the app name. For more information, see the [dotnet new](/dotnet/core/tools/dotnet-new) command in the .NET Core Guide.
+
+---
+
+### Blazor client-side authentication
+
+In Blazor client-side apps, authentication checks can be bypassed because all client-side code can be modified by users. The same is true for all client-side app technologies, including JavaScript SPA frameworks or native apps for any operating system.
+
+Implement an Authentication State Provider to integrate a Blazor client-side app with external authentication systems independently of server-side code. For more information, see the [Implement a custom Authentication State Provider](#implement-a-custom-authentication-state-provider) section.
```

Comment:
It's mostly a localization issue. In plain text (words with spaces), this can machine translate.

Is this ok? .....

> Implement an Authentication State Provider (\`AuthenticationStateProvider\`) to integrate ...